junit-platform-launcher/src/main/java/org/junit/platform/launcher/tagexpression/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * The tag expression language {@link org.junit.platform.launcher.tagexpression.Parser}
+ * and related support classes for.
```

Comment:
Since `Parser` is package-private it won't make it into the generated Javadoc. Thus, we should not link to it. I think `The tag expression language parser and related support classes.` is sufficient.

Reply:
done.